src/boss/boss.go

```diff
@@ -117,11 +122,22 @@ func BossMain() (err error) {
 		boss.autoScaler.Launch(boss.workerPool)
 	}
 
+	store, err := NewLambdaStore(Conf.Lambda_Store_Path)
```

Comment:
The Boss structure created above should be a place to store all the subsystems of the boss:
pool, lambda store, and autoscaler